README.rst

```diff
@@ -186,4 +190,9 @@ quickly identify failures and conformance rate::
 This report should be packaged with the run tests and sent to Khronos for
 conformance submission.
 
-The conformance submission requires the use of the core.csv filter.
+Conformance submission
+----------------------
+
+The conformance submission requires the use of the core.csv filter alongside
+with the run_conformance_tests.py script. Please, note that this script will
+switch on the SYCL_CTS_ENABLE_FULL_CONFORMANCE option.
```

Comment:
```suggestion
The conformance submission requires the use of the ``core.csv`` filter alongside
with the ``run_conformance_tests.py`` script. Please, note that this script will
switch on the ``SYCL_CTS_ENABLE_FULL_CONFORMANCE`` option.
```
I am unsure about the position of the last "on" in English. Should it be at the end?